config/CLIConfiguration.ts

```diff
@@ -480,7 +489,12 @@ class CLIConfiguration {
     }
 
     if (accountId) {
-      this.addOrUpdateAccount({ accountId, name: newName, env: this.getEnv() });
+      this.addOrUpdateAccount({
+        accountId,
+        name: newName,
+        env: this.getEnv(),
+        accountType: accountConfigToRename.accountType,
```

Comment:
Why do we need to include the account type here if all we're doing is renaming the existing account? Same question for `env`

Reply:
Including the account type here addresses a bug with the rename function, where it was converting all accounts to standard accounts. Specifying the accounttype and env keeps them from being overwritten.

Reply:
huh that's odd. Does that mean `addOrUpdateAccount` has unexpected behavior where it unintentionally overrides config values that aren't specified?

Reply:
If you're just merging this PR into `refactor-config` though, this doesn't have to be a blocker. Just something we should do eventually before merging the parent branch

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/192/commits/de06140671bf92e18ce4712bdbcbd2f36fac944b